sharding/contracts/validator_manager.v.py

```diff
@@ -286,176 +285,61 @@ def get_member_of_committee(
     return self.notary_pool[sampled_index]
 
 
-# Helper function to get collation header score
+# Helper function to get collation chunk root
 @public
 @constant
-def get_collation_header_score(shard_id: int128, collation_header_hash: bytes32) -> int128:
-    collation_score: int128 = convert(
-        uint256_mod(
-            convert(self.collation_headers[shard_id][collation_header_hash], 'uint256'),
-            # Mod 2^48, i.e., extract right most 6 bytes
-            convert(281474976710656, 'uint256')
-        ),
-        'int128'
-    )
-    return collation_score
+def get_collation_chunk_root(period: int128, shard_id: int128) -> bytes32:
+    return self.collation_records[period][shard_id].chunk_root
```

Comment:
Why not we just set the `collation_records` to `public`, and use its getter to access this state variable? Or does it not working well?